app/src/main/java/com/alexbar10/cryptotrack/di/DatabaseModule.kt

```diff
@@ -0,0 +1,34 @@
+package com.alexbar10.cryptotrack.di
+
+import android.content.Context
+import androidx.room.Room
+import com.alexbar10.cryptotrack.database.CryptoDatabase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class DatabaseModule {
+
+    @Provides
+    @Singleton
+    fun provideDatabase(
+        @ApplicationContext app: Context
+    ): CryptoDatabase {
+        return Room.databaseBuilder(
+            app,
+            CryptoDatabase::class.java,
+            "CRYPTO_DATABASE"
```

Comment:
As a best practice, we can make the Database name (or this kind of strings) a constant somewhere. Maybe for now you can create a constant variable inside this same file. That way if this same name is needed somewhere else we can just point to the same variable.
```
const val CRYPTO_DATABASE = "CRYPTO_DATABASE"
```

This is similar to how you used this variable in the other file -> `API_BASE_URL`